app/src/main/res/drawable/general_item_background_border_black.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <solid android:color="@color/component_color_third_party_dependency_options_list_activity_background_color" />
```

Comment:
The naming should be changed to something general so that this file can be used anywhere.

`component_color_general_item_background_solid_color`

Reply:
done.